### PR TITLE
Too large for Quenty to approve

### PR DESCRIPTION
**Pull request by Narrev**
Please review changes

### CODE REVIEW

File: Modules/qSystems/Type.lua [hunk 1]
```diff
@@ -9,40 +10,55 @@
...
+local function isAColor3(value) return pcall(set, Color3Value, "Value", value) == true end
+local function isAUDim2(value) return pcall(set, FrameValue, "Position", value) == true end
+local function isAVector3(value) return pcall(set, Vector3Value, "Value", value) == true end
+local function isAUDim(value) return pcall(function() return UDim() + value end) == true end
+local function isAVector2(value) return pcall(function() return Vector2() + value end) == true end
+local function isACoordinateFrame(value) return pcall(set, CFrameValue, "Value", value) == true end
+local function isABrickColor(value)	return pcall(set, BrickColor3Value, "Value", value) == true end
+local function isPositiveInt(number) return type(number) == "number" and number > 0 and floor(number) == number end
+
+local function coerceIntoEnum(value, enum)
```
Comment (Narrev) [inline Modules/qSystems/Type.lua]: N

**Comment by Kinrany** [comment]
Maybe try splitting it into several independent pull requests. This way @Quenty will be able to accept/deny each of them separately and won't have to go over over 1k changes at once.

**Comment by Narrev** [comment]
Thank you for the advice on a 6 month old pull request, but there is no way to deprecate qSystems and put things in their proper place without a lot of movement.

**Comment by Kinrany** [comment]
I see. You could leave a description of your changes then, so that Quenty knows what is it you're trying to do and how.

And there are some cosmetic changes that can be removed from this pull request. There are lots of lines where only whitespaces were changed, for example.

**Comment by Quenty** [comment]
Depreciating qSystems by moving methods into other libraries and then proxying them into qSystems will be acceptable.

**Pull request closed without merging**